saleor/wsgi/__init__.py

```diff
@@ -27,3 +28,16 @@
 # from helloworld.wsgi import HelloWorldApplication
 # application = HelloWorldApplication(application)
 application = health_check(application, "/health/")
+
+# Warm-up the django application instead of letting it lazy-load
+application(
+    {
+        "REQUEST_METHOD": "GET",
+        "SERVER_NAME": "127.0.0.1",
```

Comment:
This could fail before fully warming up if the name does not match ALLOWED_HOSTS.